functions/_tide_item_context.fish

```diff
@@ -1,12 +1,20 @@
 function _tide_item_context
+    set -l displayedHost $hostname
+
+    if test "$tide_short_hostname" = 'true'
```

Comment:
This variable should be `tide_context_short_hostname`

Reply:
Variable should also be added to the wiki documentation, though of course that can't be in the PR 😞